core-web/.eslintrc.json

```diff
@@ -26,6 +26,14 @@
             "files": ["*.ts", "*.tsx"],
             "extends": ["plugin:@nx/typescript"],
             "rules": {
+                "@typescript-eslint/no-inferrable-types": "off",
+                "@typescript-eslint/no-empty-function": "off",
```

Comment:
WHAT!!!!! Don't like this. Why are we doing this?

Reply:
this:
 `"@typescript-eslint/no-inferrable-types"`: "off", touch several files, every variable/input in the codebase with an assigned value wants to be rid of the type because it is inferred.
and this:

Reply:
Fixed the files and removed the rules.